code/modules/recycling/conveyor2.dm

```diff
@@ -246,9 +281,6 @@
 			S.update()
 		CHECK_TICK
 

```

Comment:
Maybe...
On one hand, people could easily make conveyors with multiple switches, including one-way only independent ones - assuming I do the multitool thing (which I do want to).
On the other hand, people might like to do something clever, where some levers only activate some segments. I can't imagine what though.